### PR TITLE
Single clean commit for parallel ground

### PR DESCRIPTION
**Pull request by jlaura**
This is a single, clean commit PR, that adds ground. I tried to incorporate all past comments into this. I will try to get some tests added shortly.

### CODE REVIEW

File: autocnet/matcher/ground.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+    with ncg.session_scope() as session:
+        # Check to see if the point already exists
+        g = Points(path=ground_mosaic.file_name,
+                    choosername='find_most_interesting_ground',
+                    aprioriline=line,
+                    apriorisample=sample,
+                    line=newline,
+                    sample=newsample,
+                    geom=p,
+                    ignore=False)
```
Comment (ladoramkershner) [inline autocnet/matcher/ground.py:253]: shouldn't this be appending to the CandidateGroundPoints database model?

File: autocnet/matcher/ground.py [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+        return
+
+    left_x, _, top_y, _ = image.image_extent
+    newsample = left_x + interesting.x
+    newline = top_y + interesting.y
+
+    # @LAK - this needs eyeballs to confirm correct oc/og
+    newpoint = isis.point_info(ground_mosaic.file_name, newsample, newline, 'image')
+    p = Point(newpoint[0].get('PositiveEast360Longitude'),
+              newpoint[0].get('PlanetocentricLatitude'))
```
Comment (ladoramkershner) [inline autocnet/matcher/ground.py:242]: This looks good to me! ISIS operates in ocentric, the reason we needed to switch to ographic is when we were working with pyproj to convert lat/lon to/from BCBF.

File: autocnet/matcher/ground.py (hunk 1 above)
Comment (jlaura) [reply autocnet/matcher/ground.py:253]: Updated! Good catch. :)

**Comment by jlaura** [comment]
@ladoramkershner Updated based on your comment (locally; yesterday) and then synced the changes (today). I think this should be good to go.

File: autocnet/io/db/model.py [hunk 3]
```diff
@@ -443,7 +477,7 @@ def try_db_creation(engine, config):
     Points.rectangular_srid = rectangular_srid
     Points.semimajor_rad = spatial['semimajor_rad']
     Points.semiminor_rad = spatial['semiminor_rad']
-    for cls in [Points, Overlay, Images, Keypoints, Matches]:
+    for cls in [Points, Overlay, Images, Keypoints, Matches, CandidateGroundPoints()]:
```
[the remark this answers is not in the document]
Comment (jlaura) [reply autocnet/io/db/model.py]: Fixed this - I have no idea why I included an instance and not a class...amazing - it works either way. sqlalchemy must be doing some introspection that works either way.

**Pull request merged**